subsys/bluetooth/audio/shell/gmap.c

```diff
@@ -0,0 +1,564 @@
+/** @file
+ *  @brief Bluetooth Gaming Audio Profile shell
+ *
+ */
+
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+
+#include <zephyr/bluetooth/audio/gmap_lc3_preset.h>
+#include <zephyr/bluetooth/audio/gmap.h>
+#include <zephyr/kernel.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+
+#include "shell/bt.h"
+#include "audio.h"
+
+#define UNICAST_SINK_SUPPORTED (CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT > 0)
+#define UNICAST_SRC_SUPPORTED  (CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT > 0)
+
+#define GMAP_AC_MAX_CONN   2U
```

Comment:
Should this be a config or will it always be 2?

Reply:
It will always be 2 based on the defined audio configurations from the GMAP spec